File: bionic-js-tool/testing-code/swift/project-with-host-files/HostProject/host/FerrariCalifornia.swift
import JavaScriptCore
import Bjs

class FerrariCalifornia: MotorVehicle {
    
    private static var _bjsLocator: BjsLocator = BjsLocator("BeautifulVehicles", "FerrariCalifornia")
    override class var bjsLocator: BjsLocator { _bjsLocator }
    override class func bjsFactory(_ jsObject: JSValue) -> FerrariCalifornia { FerrariCalifornia(jsObject) }
}

File: bionic-js-tool/testing-code/java/project-with-host-files/HostProject/src/main/java/test/project/host/FerrariCalifornia.java
package test.project.host;

import jjbridge.api.runtime.JSReference;
import bionic.js.Bjs;
import bionic.js.BjsTypeInfo;
import bionic.js.BjsObjectTypeInfo;
import test.project.host.libs.MotorVehicle;

@BjsTypeInfo.BjsLocation(project = "BeautifulVehicles", module = "FerrariCalifornia")
public class FerrariCalifornia extends MotorVehicle {
    
    private static final JSReference bjsClass = BjsObjectTypeInfo.get(FerrariCalifornia.class).bjsClass();
    public static final Bjs bjs = BjsObjectTypeInfo.get(FerrariCalifornia.class).bjsLocator.get();
    public static final Bjs.JSReferenceConverter<FerrariCalifornia> bjsFactory = FerrariCalifornia::new;
}